### DISCUSSION TITLE
Newbie questions about how to use this reader

### DISCUSSION THREAD

**Discussion by ovizii** [edrlab/thorium-reader#2644] [Q&A]
I read the Readme and saw instructions on how to run it. They stated it needs NPM and NodeJS. Seeing there’s a Dockerimage and a docker.sh script I was wondering whether it is possible to run this manually self-hosted via Docker and access it via a web interface? 

Let's leave aside questions about reverse proxies and authentication/authorization for now.

**Comment by danielweck** [comment]
Thorium is a desktop application, you can download ready-made "installers" from: https://github.com/edrlab/thorium-reader/releases

**Reply by zhen-huan-hu** [reply]
Would be nice to be able to run Thorium Reader on a web server though (as a Docker app), like what VS Code for the Web does.
